server.js

```diff
@@ -7,10 +7,12 @@ var express = require('express');
 var app = express();
 
 app.use(express.static(__dirname + '/'));
+console.debug(`Dirname is ${__dirname}`)
```

Comment:
could probably get rid of this 🐛

Reply:
Thanks for the catch! Now removed from code.